packages/react/src/components/sectional-banner/sectional-banner.tsx

```diff
@@ -75,20 +78,42 @@ function abstractContainer(
         ${(props) => focusVisibleReset(props, true)};
 
         ${BannerIcon} {
-            color: ${(props) => (iconColor ? props.theme.notifications[iconColor] : props.theme.main['primary-3'])};
+            /* eslint-disable-next-line stylistic/declaration-colon-newline-after */
+            color:
+                ${(props) => (iconColor ? props.theme.component[iconColor]
+                    : props.theme.component['sectional-banner-icon-color'])};
             flex: 0 0 auto;
             height: 1rem;
             width: 1rem;
         }
     `;
 }
 
-const NeutralContainer = abstractContainer('#fafafa', 'neutral-1.1');
-const InfoContainer = abstractContainer('#f3f9fd', 'info-1.1');
-const DiscoveryContainer = abstractContainer('#f9f7fb', 'discovery-1.1');
-const SuccessContainer = abstractContainer('#f6fbf8', 'success-1.1');
-const WarningContainer = abstractContainer('#fffbf5', 'warning-3.1', 'warning-3.4');
-const AlertContainer = abstractContainer('#fdf6f7', 'alert-2.1');
+const NeutralContainer = abstractContainer(
+    'sectional-banner-container-neutral-background-color',
+    'sectional-banner-container-neutral-border-color',
+);
```

Comment:
Ce serait pas mieux sans le `container`?
```suggestion
const NeutralContainer = abstractContainer(
    'sectional-banner-neutral-background-color',
    'sectional-banner-neutral-border-color',
);
```